dbt/adapters/databricks/python_submissions.py

```diff
@@ -86,6 +86,9 @@ def insert_command(cls, command: CommandExecution) -> None:
 
     @classmethod
     def cancel_runs(cls, session: Session) -> None:
+        # TODO: let Databricks Workflows handle timeouts if a timeout is defined
```

Comment:
Not sure if there's a good way to fit this in with the whole dbt workflow. My main use-case was the initial run/backfilling lots of data, but I think just pressing "play" on the job by hand and getting out of the dbt limitations would be good enough.